### PR TITLE
Use Keyword instead of SurveyKeyword

### PR DESCRIPTION
**Pull request by gcapalbo**
Updates all messaging references to keywords to use the ORM model rather than the Couch model.

Opening for review only, don't merge yet - I need to run the sync in all environments first and do a little bit of testing.

@emord

### CODE REVIEW

**Comment by gcapalbo** [comment]
code buddies @orangejenny @biyeun

File: custom/apps/wisepill/views.py [hunk 1]
```diff
@@ -46,15 +46,13 @@ def device_data(request):
...
-                for survey_keyword_action in survey_keyword.actions:
-                    if survey_keyword_action.action == METHOD_STRUCTURED_SMS:
-                        handle_structured_sms(survey_keyword, survey_keyword_action, case, None, "DEVICE_EVENT,%s" % data, send_response=False)
-                        event.processed = True
-                        event.save()
-                        break
+        for survey_keyword in Keyword.get_by_domain(case.domain).filter(keyword__iexact="DEVICE_EVENT"):
+            for survey_keyword_action in survey_keyword.keywordaction_set.all():
+                if survey_keyword_action.action == KeywordAction.ACTION_STRUCTURED_SMS:
+                    handle_structured_sms(survey_keyword, survey_keyword_action, case, None, "DEVICE_EVENT,%s" % data, send_response=False)
```
Comment (dimagimon) [inline custom/apps/wisepill/views.py]: E501 line too long (139 > 115 characters)

**Comment by gcapalbo** [comment]
the sync has been run in all environments, and the code updates have been tested - this is ready to go from my end

File: corehq/apps/reminders/views.py [hunk 2]
```diff
@@ -492,7 +492,7 @@ class AddStructuredKeywordView(BaseMessagingSectionView):
     template_name = 'reminders/keyword.html'
     process_structured_message = True
 
-    @method_decorator(requires_privilege_with_fallback(privileges.OUTBOUND_SMS))
+    @method_decorator(requires_privilege_with_fallback(privileges.INBOUND_SMS))
```
Comment (orangejenny) [inline corehq/apps/reminders/views.py]: Curious, why did this change?

Comment (gcapalbo) [inline corehq/apps/reminders/views.py]: @orangejenny it was actually wrong, these views shouldn't be seen unless the project has access to inbound sms

Comment (orangejenny) [inline corehq/apps/reminders/views.py]: cool, thanks for explaining

**Review by orangejenny — APPROVED** [review]

**Pull request merged**